.env.example

```diff
@@ -1 +1,2 @@
 TELEGRAM_BOT_API_TOKEN=token
+ENVIRONMENT=development
```

Comment:
You should set up your editor to add an empty line at the end of the file.
Of course, it's not super important thing, but it's kinda a rule.

origins: https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-file

also you should check and add https://editorconfig.org/ if it seems valuable for you.